content/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh.md

```diff
@@ -0,0 +1,333 @@
+
+However, this evolution presents challenges for monitoring, as applications and services are inherently more distributed in this environment. Having a good application performance monitoring (APM) tool is becoming essential within the hybrid cloud environment. It can consolidate performance metrics and troubleshoot data for assets across the hybrid cloud environment into a single application. It makes it easier to track metrics and allows the enterprise to resolve problems quickly.
+
+For this blog post, I c﻿hose Apache SkyWalking as an APM tool and describe how to set up it, as a *self-hosted* APM tool, for monitoring and alerting the application performance across the hybrid cloud environment. I took the service mesh as an auto instrumentation mechanism for application monitoring, without adding any manual process to instrument existing applications. 
+
+## Apache SkyWalking
+
+[Apache SkyWalking](https://skywalking.apache.org/) is an open source APM tool that can monitor, trace, and diagnose distributed systems. It’s especially designed for microservices, cloud native and container-based architectures. 
+
+Apart from a list of supported agents to be used for instrumenting applications, Apache SkyWalking implements an [Envoy Access Log Service (ALS)](https://www.envoyproxy.io/docs/envoy/v1.18.2/api-v2/service/accesslog/v2/als.proto) based solution to provide observability on the service mesh under Kubernetes environment, no matter the architecture or language. A service mesh provides a mesh of _Layer 7_ proxies that manages network traffic between services. It supports application observability at the platform layer, instead of the application layer, by abstracting away how inter-process and service to service communications being handled in Kubernetes. Using a list of implemented analyzers, e.g.,  _k8s-mesh_ and _mx-mesh_, Apache SkyWalking can receive and analyze the detailed access logs of all requests, both _**HTTP**_ and _**TCP**_, emitted from Envoy ALS.  With this solution, users could get the application service topology map, metrics graph, request details and error message with a very nice visualization. This observation solution is much easier to be added without language-specific technology. It can be extremely important for monitoring and visualizing applications that consist of many microservices running across on-premises, cloud-based or hybrid environments. 
+
+Apache SkyWalking is lightweight and scalable. It can be easily set up as a *self-hosted* APM tool within a hybrid cloud environment, without any additional external resources for hosting the tool. This can help in the context of a resource-constrained environment and remove the privacy and security concerns customers may have in putting customer data that goes out to third-party services.   
+
+## Set up Apache SkyWalking
+
+### Prerequisites
+
+Before you start, make sure you have the following required elements:
+
```

Comment:
In these bullets, please remove the semi-colons. You can use a period at the end of each because you have a number of them that have more than one line/sentence in them, like you did in the other blog post in this series. But don't use semi-colons.